troposphere/static/js/collections/InstancePlaybookCollection.js

```diff
@@ -0,0 +1,20 @@
+import InstanceCollection from "./InstanceCollection";
+import InstancePlaybook from "models/InstancePlaybook";
+import globals from "globals";
+
+export default InstanceCollection.extend({
+    model: InstancePlaybook,
+
+    url: globals.API_V2_ROOT + "/instance_playbooks",
```

Comment:
Hmm... This is pretty specific to the implementation (because ansible-playbooks can take a while to start running, and can return failure, there is a use-case for the user to know whats happening "behind the curtain") but if we wanted to avoid showing our hand, I guess we could call it `InstanceCommandStatus`?